src/components/shared/Slider/Slider.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Slider = ({ min, max, step, defaultValue, handleChange }) => {
```

Comment:
What do you think about using default values?

```suggestion
export const Slider = ({ min = 0, max = 100, step = 1, defaultValue = 0, handleChange }) => {
```

Reply:
If I use default values it's doesn't have sense use the original API. I'm not sure if it's necessary to use default values.